#Request data
sensor_msgs/PointCloud2 input_pc
---
#Response data
string info # for mission result
